### PR TITLE
Save Network Keys in Data Directory

### PR DESCRIPTION
**Pull request by nisdas**
This allows our ecdsa private key which is used by both discv5 and libp2p to be saved into our data directory. This ensures that after your first run with prysm, your private key doesn't change and prysm
reads the data directory by default to find the network private key.

- [x] Allow Network Keys to be saved in data directory

### CODE REVIEW

File: beacon-chain/node/node.go [hunk 1]
```diff
@@ -226,6 +226,7 @@ func (b *BeaconNode) registerP2P(ctx *cli.Context) error {
 		StaticPeers:       sliceutil.SplitCommaSeparated(ctx.GlobalStringSlice(cmd.StaticPeers.Name)),
 		BootstrapNodeAddr: bootnodeENR,
 		RelayNodeAddr:     ctx.GlobalString(cmd.RelayNode.Name),
+		BaseDataDir:       ctx.GlobalString(cmd.DataDirFlag.Name),
```
Comment (terencechain) [inline beacon-chain/node/node.go]: What does "base" data imply here? Maybe "p2p" data or "network" data? Better to be explicit

**Comment by prestonvanloon** [comment]
Please add description. What is this feature?

File: beacon-chain/node/node.go (hunk 1 above)
Comment (prestonvanloon) [reply beacon-chain/node/node.go]: Why not call it DataDir?

Comment (rauljordan) [reply beacon-chain/node/node.go]: Resolved

**Pull request merged**